packages/react-app-revamp/hooks/useContest/index.ts

```diff
@@ -294,11 +294,12 @@ export function useContest() {
         setSubmitProposalTokenAddress(results[4]);
         setSubmitProposalToken(votingTokenRawData);
       }
+      await checkIfCurrentUserQualifyToVote();
+
       if(accountData?.address) {
         // Current user votes
         await updateCurrentUserVotes();
         // Check snapshot
```

Comment:
Just a small thing, do you mind if we move this to above where the function call now is?